enterprise/cmd/executor/internal/command/util.go

```diff
@@ -25,3 +30,19 @@ func intersperse(flag string, values []string) []string {
 
 	return interspersed
 }
+
+// quoteEnv returns a slice of env vars in which env vars that contain a whitespace have been quoted
+func quoteEnv(env []string) []string {
+	quotedEnv := make([]string, len(env))
+
+	for i, e := range env {
+		if strings.Contains(e, " ") {
+			elems := strings.SplitN(e, "=", 2)
+			quotedEnv[i] = fmt.Sprintf(`%s="%s"`, elems[0], elems[1])
```

Comment:
```suggestion
			quotedEnv[i] = fmt.Sprintf(`%s=%q`, elems[0], elems[1])
```

? 

Otherwise, what happens when the env var arg contains a `"` 🤓:
https://go.dev/play/p/ni3rzgiRXSr

![image](https://user-images.githubusercontent.com/19534377/151971410-49bc1230-cbc1-4c55-8948-f6a805d77e65.png)

Reply:
This looks like a glaring security concern :upside_down_face:

Reply:
I had this at first but then changed it for some reason (I think I made a typo in the test). Will change back.

Reply:
can we also add a test to ensure it's not allowing to inject additional params?

Reply:
Done